src/quizzes/chapter-3/3.4/docker-quiz.js

```diff
@@ -0,0 +1,60 @@
+const rawQuizdown = `
+---
+shuffleAnswers: true
+---
+
+# Assume you made a Dockerfile that, when built into an image and ran, will install a handful of packages and log a success string. If you are on MacOS and your friend is on Windows, can you send them the image and expect the container to build and operate exactly the same?
+
+1. [x] Yes
+	> Excellent, this is one of the main points of Docker and containers as a whole! You can share images between different environments and they *will* behave the same. They greatly help relieve the "It works on my machine" allegations
```

Comment:
_mostly_ I think you are right on the main point but different CPU architecture does throw a wrench in this. Like OOTB I do not think you will be able to run an ARM image on an x86 Windows box. The only reason we can run x86 images on MacOS is due to the Rosetta emulation layer.

Reply:
I think this is the case but again I think high level you are right and this is an edge case that they can learn about later